.github/workflows/api-tests-with-private-packages.yml

```diff
@@ -42,7 +44,35 @@ jobs:
 
       - name: Install Dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-        run: make install-packages
+        shell: bash
+        run: |
+          echo "https://${{ secrets.GH_PRIVATE_ACCESS_TOKEN }}:@github.com" > ${HOME}/.git-credentials
+          git config --global credential.helper store
+          make install-packages opts="--with saml,auth-controller,workflows"
+          rm -rf ${HOME}/.git-credentials
+
+      - name: Install SAML Dependencies
+        run: sudo apt-get install -y xmlsec1
+
+      - name: Get workflows-logic package version
+        id: workflow-version
+        run: |
+          tag=$(grep -A 1 "\[tool.poetry.group.workflows.dependencies\]" pyproject.toml | awk -F '"' '{printf $4}')
```

Comment:
I'd rather use something like https://github.com/benc-uk/workflow-dispatch to trigger the workflow tests in their own repo. Actually, this might be a better solution for all the private modules overall

Reply:
Because each package runs only its own tests, this will not help us cover the use case where updating package X breaks package Y.